app/transforms/utc.js

```diff
@@ -0,0 +1,12 @@
+import moment from 'moment';
+import DS from "ember-data";
+
+export default DS.Transform.extend({
+  serialize: function(value) {
+    return value ? value.toJSON() : null;
+  },
+
+  deserialize: function(value) {
+    return moment.utc(value);
```

Comment:
Challenge here could be supporting different timezones. If we let the client convert to utc, the date/times passed by fhir may be misrepresented in the client. Given a date `2016-12-31T17:00:00-0700`, `moment.utc('2016-12-31T17:00:00-0700')` will misrepresent the date as `Jan 1 2017`. I would prefer to display the date passed by fhir as is - possibly using `moment.parse(<dateTime>)` to prevent the client from handling timezone conversions. Thoughts and discussions are welcome